src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Completion/DirectiveCompletionItemProvider.cs

```diff
@@ -145,6 +150,10 @@ internal static ImmutableArray<RazorCompletionItem> GetDirectiveCompletionItems(
                 completionDisplayText,
                 directive.Directive,
                 RazorCompletionItemKind.Directive,
+                // Make sort text one less than display text so if there are any delegated completion items
+                // with the same display text in the combined completion list, they will be sorted below
+                // our items.
+                sortText: completionDisplayText[..Math.Max(1, completionDisplayText.Length - 1)],
```

Comment:
Is this setting the sort text for `using` to `usin`? isn't that a bit weird if there is a C# variable in scope called `usine`?

I'm not sure what a better option here is though...

Reply:
Yes, so that C# "using" snippet is always after. 

I guess since I added snippet re-writer anyway, I could make C# "using" directive sort text to be "using " (using + <space>). How does that sound? Seems like that might work better (though more messing C# items).

Reply:
Tha sounds good to me

Reply:
Seems to work, so I'm going with that solution

<img width="260" alt="image" src="https://github.com/dotnet/razor/assets/1863656/02d2b864-44fc-435a-9a2a-cbfbcf8ae334">